Neos.ContentRepository.Core/Classes/Feature/NodeDuplication/NodeDuplicationCommandHandler.php

```diff
@@ -52,6 +54,11 @@ public function __construct(
     ) {
     }
 
+    protected function getContentGraph(WorkspaceName $workspaceName, CommandHandlingDependencies $commandHandlingDependencies): ContentGraphInterface
+    {
+        return $commandHandlingDependencies->getContentGraph($workspaceName);
+    }
```

Comment:
why were these kind of methods added to multiple command handler? i think its a leftover piece?

Reply:
removed